src/main/java/frc/robot/RobotContainer.java

```diff
@@ -10,8 +10,9 @@
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.Command;
-import frc.robot.subsystems.ExampleSubsystem.ExampleSubsystem;
-import frc.robot.subsystems.ExampleSubsystem.commands.ExampleCommand;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import frc.robot.subsystems.Shooter.Shooter;
+import frc.robot.subsystems.Shooter.commands.PowerShoot;
```

Comment:
rename the package `Shooter` to `shooter`. packages name shouldn't be capitalized